scripts/build.js

```diff
@@ -138,6 +138,16 @@ function build(previousSizeMap) {
       console.log('  ' + chalk.cyan('git') + ' push -f origin gh-pages');
       console.log('  ' + chalk.cyan('git') + ' checkout -');
       console.log();
+      console.log('To publish it at ' + chalk.green(homepagePath) + ' using the docs/ folder, run:');
+      console.log();
+      console.log('  ' + chalk.cyan('git') + ' commit -am ' + chalk.yellow('"Save local changes"'));
+      console.log('  ' + chalk.cyan('rm') + ' -r docs/');
+      console.log('  ' + chalk.cyan('mv') + ' build/ docs/');
+      console.log('  ' + chalk.cyan('git') + ' add -f docs');
+      console.log('  ' + chalk.cyan('git') + ' commit -am ' + chalk.yellow('"Rebuild website"'));
+      console.log('  ' + chalk.cyan('git') + ' filter-branch -f --prune-empty --subdirectory-filter docs');
```

Comment:
Do we need this command? I think it's only necessary for `gh-pages` because we needed to make a nested folder to be pushed as top-level.